modules/ROOT/pages/building-https-proxy.adoc

```diff
@@ -3,62 +3,49 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-API Manager 2.x enables you to use the secrets you stored in your secret group to build an HTTPS-based API for your CloudHub or hybrid deployments. +
+API Manager 2.x enables you to use the secrets you stored in your secret group to build an HTTPS-based API for your CloudHub or hybrid deployments. 
 By configuring a TLS context for your inbound and outbound traffic, you can ensure your traffic's security.
 
-The secrets manager integration is only supported by API Proxies running Mule 4, and only for CloudHub, Hybrid, and standalone deployments. +
+The secrets manager integration is only supported by API Proxies running Mule 4, and only for CloudHub, Hybrid, and standalone deployments.
 Supported keystores for the TLS context are JCEKS, PKCS12, and JKS.
 
-If you want to build an HTTPS Proxy in Mule 3 or Runtime Fabric deployment target, follow the xref:mule-runtime::build-an-https-service.adoc[Build an HTTPS Service] tutorial.
+To build an HTTPS Proxy in Mule 3 or Runtime Fabric deployment target, see xref:mule-runtime::build-an-https-service.adoc[Build an HTTPS Service].
 
-== Prerequisites
+== Before You Begin
 
-To configure your HTTPS API Proxy using secrets stored in your secret group, you need to meet the following requirements:
+To configure your HTTPS API Proxy using secrets stored in your secret group, ensure you have:
 
-* Secrets Manager Permissions enabled for your user. +
+* Requested the following permissions from your Anypoint Platform admin:
+** Secrets Manager permissions +
 To configure an existing TLS context for the API Proxy, you need the *Read secrets metadata* permission.
 Review the xref:anypoint-security::asm-permission-concept.adoc[Secrets Manager Permissions] reference for more information.
-* Make sure your user has *Manage API Configuration* role.
-* A secret group along with a TLS context in Secrets Manager. +
-Follow the xref:anypoint-security::asm-secret-group-creation-task.adoc[Create a Secret Group] instructions along with the  xref:anypoint-security::asm-secret-group-creation-task.adoc#add-a-tls-context[Add a TLS Context] task contained in that article if you need help creating a TLS context. +
-Make sure that your TLS context's target is of type "Mule", and that the Secret Group is marked as "downloadable".
-+
-If you are not familiar with the secrets manager concepts, you should also review the xref:anypoint-security::asm-secret-group-concept.adoc[Secrets Group Basic Concepts].
+** API Manager permissions +
+To apply a TLS context to an API proxy in API Manager, you need the *Manage API Configuration* role.
+See the xref:access-management::permissions-by-product.adoc#api-manager[API Manager] section of the Permissions
+by Product page for more information.
```

Comment:
Shouldn't it be in line 24? 

I'm not sure how it looks with the line break.

Reply:
The line break shouldn't matter here.